discord/app/context.py

```diff
@@ -35,7 +35,8 @@ class InteractionContext:
     .. versionadded:: 2.0
     """
 
-    def __init__(self, interaction: Interaction):
+    def __init__(self, bot, interaction: Interaction):
```

Comment:
I was having trouble importing `discord.Bot` here for the type hint without hitting a circular import.